client/src/services/formulas.js

```diff
@@ -33,4 +33,13 @@ const getFormulaAttributes = (formulaId) => {
   return formulaAttributes;
 };
 
-export default { getFormulas, setFormula, getFormulaName, getFormulaAttributes };
+// A function for getting the displayable label from a label key
+// for example: the label key maxPoints would return Max Points
+const getAttributeLabel = (labelKey) => {
```

Comment:
Not a mandatory change!! But I wonder if this should be in text format services or something since it's not specific to just formulas even if it is used just here.